### ISSUE TITLE
--experimental-scope-hoisting breaks project using rebass

### ISSUE THREAD

**Issue by midnightcodr** [parcel-bundler/parcel#2977]
**Choose one:** a 🐛 bug report

### 🎛 Configuration (.babelrc, package.json, cli command)
Please see https://gist.github.com/midnightcodr/fbcfc63ab80951b3f13b330b0ec1f406

```js
{
  "presets": [
    "env",
    "react"
  ],
  "plugins": [
    "transform-object-rest-spread",
    "transform-class-properties"
  ]
}
```

### 🤔 Expected Behavior

If I build the project without --experimental-scope-hoisting, everything works as expected. But with the option on, when serving the site, I am getting

![screenshot](https://user-images.githubusercontent.com/1152119/57103438-bd485e00-6cf3-11e9-8e3d-23f90ef078ba.png)

In the browser console and a blank page.

### 😯 Current Behavior

It should render a page with "Hello React!" and no js error.

### 💻 Code Sample

Please see https://gist.github.com/midnightcodr/fbcfc63ab80951b3f13b330b0ec1f406

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           |1.12.3
| Node             |10.15.3
| npm/Yarn         |6.4.1
| Operating System |Ubuntu 19.04

**Comment by devongovett** [comment]
This is due to a missing `__esModule` flag that babel uses for interop between common JS and ES6 modules. I made a commit to the Parcel 2 tree shaking branch that fixes this: https://github.com/parcel-bundler/parcel/pull/2967/commits/ab98afe2ba3dcf2a8978580cd119ded770212fc5. Will need to be back ported.

**Comment by devongovett** [comment]
Should be fixed by #2993.

**Comment by midnightcodr** [comment]
How can I start using the version with your fix @devongovett ?
